src/tab.group.test.interactions.ts

```diff
@@ -217,6 +224,43 @@ it('does not have overflow buttons when not overflowing', async () => {
   expect(endButton).to.be.null;
 });
 
+it('hides its overflow buttons when overall tab content is reduced', async () => {
+  // The selected tab indicator transitions its width and position. The transitions
+  // are disabled to simplify and speed up the test.
+  await emulateMedia({ reducedMotion: 'reduce' });
+
+  const host = await fixture(html`
+    <glide-core-tab-group>
+      <glide-core-tab slot="nav" panel="1">${'x'.repeat(500)}</glide-core-tab>
+      <glide-core-tab slot="nav" panel="2">Two</glide-core-tab>
+      <glide-core-tab slot="nav" panel="3">Three</glide-core-tab>
+
+      <glide-core-tab-panel name="1">One</glide-core-tab-panel>
+      <glide-core-tab-panel name="2">Two</glide-core-tab-panel>
+      <glide-core-tab-panel name="3">Three</glide-core-tab-panel>
+    </glide-core-tab-group>
+  `);
+
+  await waitUntil(() => {
+    return (
+      host.shadowRoot?.querySelector('[data-test="overflow-start-button"]') &&
+      host.shadowRoot?.querySelector('[data-test="overflow-end-button"]')
+    );
+  });
+
+  const firstTab = host.querySelector('glide-core-tab');
+
+  assert(firstTab);
+  firstTab.innerHTML = 'One';
+
+  await waitUntil(() => {
```

Comment:
It always feels odd not having an expectation. But the expectation would be exactly this.